package.json

```diff
@@ -1,8 +1,10 @@
 {
     "name": "botbuilder-tools",
     "scripts": {
-        "test:travis": "nyc mocha \"@(Chatdown|Ludown|LUIS|QnAMaker)*/test/*.test.suite.js\" --timeout 100000 && nyc report --reporter=text-lcov | coveralls",
-        "eslint:travis": "eslint --format \"node_modules/eslint-friendly-formatter\" \"./**/*.js\" \"./@(Dispatch|LUIS|LUISGen|QnAMaker)/bin/*\""
+        "postinstall": "(cd Chatdown && npm install); (cd Dispatch && npm install); (cd Ludown && npm install); (cd LUIS && npm install); (cd LUISGen && npm install); (cd MSBot && npm install); (cd QnAMaker && npm install)",
```

Comment:
This is NPM specific and will run each time the end user installs these tools. This should not be necessary if we're publishing correctly.